src/app/(Authentication)/SignUp/Email/index.tsx

```diff
@@ -42,10 +43,17 @@ export default function SignUpScreen() {
     }
```

Comment:
Does our `validateEmail()` function check against the public users table? We could reduce a lot of OTP checks from this check alone.

Reply:
not right now I believe. I guess I should make a check yeah

Reply:
revisit tmrw, did a user check.